rimage/elf.c

```diff
@@ -89,18 +89,18 @@ static int elf_read_sections(struct image *image, struct module *module,
 		module->fw_ready_index = -EINVAL;
 	} else {
 		/* find manifest module data */
-		module->bss_index = elf_find_section(image, module, ".bss");
```

Comment:
Typo in commit message: "Unused arguments _shouldn't_ be passed to function".

Reply:
changed